PMSCore/Utils/UserManager.cs

```diff
@@ -0,0 +1,138 @@
+﻿using Microsoft.Extensions.Logging;
+using PMSCore;
+
+namespace PMSCore.Utils
+{
+    internal class UserManager
```

Comment:
this class should be a singletone we only need a user manager instance in the entire project scope